rdflib 6.0.0 does not always return bytes, breaking extruct.rdfa.RDFaExtractor

When installing `extruct` from pypi, I get failing tests because `extruct` depends on `PyRDFA`, which itself depends on `rdflib`. 

`rdflib` was recently upgraded to version `6.0.0`, and changed `Graph.serialize` to return bytes or strings, depending on the input parameters:
```
g.serialize(
    destination: Union[str, io.BufferedIOBase, NoneType] = None,
    format: str = 'turtle',
    base: Optional[str] = None,
    encoding: Optional[str] = None,
    **args,
) -> Union[bytes, str, NoneType]
Docstring:
Serialize the Graph to destination

If destination is None serialize method returns the serialization as
bytes or string.

If encoding is None and destination is None, returns a string
If encoding is set, and Destination is None, returns bytes

Format defaults to turtle.

Format support can be extended with plugins,
but "xml", "n3", "turtle", "nt", "pretty-xml", "trix", "trig" and "nquads" are built in.
```

Here's my failing test output:  

```python
src/gourmand/plugins/import_export/website_import_plugins/schema_org_parser.py:36: in preparse
    self.data = scrape_schema_recipe.scrape(self.text, python_objects=True)
venv/lib/python3.9/site-packages/scrape_schema_recipe/scrape.py:103: in scrape
    data = extruct.extract(location)
venv/lib/python3.9/site-packages/extruct/_extruct.py:108: in extract
    output[syntax] = list(extract(document, base_url=base_url))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <extruct.rdfa.RDFaExtractor object at 0x7fedc96b05b0>, document = <Element html at 0x7fedc96ab770>, base_url = None, expanded = True

    def extract_items(self, document, base_url=None, expanded=True):
        options = Options(output_processor_graph=True,
                          embedded_rdf=False,
                          space_preserve=True,
                          vocab_expansion=False,
                          vocab_cache=False,
                          vocab_cache_report=False,
                          refresh_vocab_cache=False,
                          check_lite=False)
        g = PyRdfa(options, base=base_url).graph_from_DOM(document, graph=Graph(), pgraph=Graph())
>       jsonld_string = g.serialize(format='json-ld', auto_compact=not expanded).decode('utf-8')
E       AttributeError: 'str' object has no attribute 'decode'

venv/lib/python3.9/site-packages/extruct/rdfa.py:154: AttributeError
```

I'm happy to do a fix for this, but it can go two ways: either pinning `rdflib` to the previous `5.0.0` version, or removing the `decode` here (with the given parameters, we will always get a string back).  
Which one would you prefer?

Thanks!

--- a/extruct/rdfa.py
+++ b/extruct/rdfa.py
@@ -151,7 +151,11 @@ class RDFaExtractor(object):
                           refresh_vocab_cache=False,
                           check_lite=False)
         g = PyRdfa(options, base=base_url).graph_from_DOM(document, graph=Graph(), pgraph=Graph())
-        jsonld_string = g.serialize(format='json-ld', auto_compact=not expanded).decode('utf-8')
+        jsonld_string = g.serialize(format='json-ld', auto_compact=not expanded)
+
+        # rdflib may return either bytes or strings
+        if isinstance(jsonld_string, bytes):
+            jsonld_string = jsonld_string.decode('utf-8')
         
         try:
             # hack to fix the ordering of multi-value properties (see issue 116)

